pkg/bootstrap/docker/run/run.go

```diff
@@ -46,6 +47,12 @@ func (h *Runner) Name(name string) *Runner {
 	return h
 }
 
+// UserNamespace sets user namespace mode flag that affects behavior of other options.
```

Comment:
I would like to make this part of the constructor for this helper (NewRunHelper). I don't think we should rely on the client to set it to the proper value.